pages/partials/title.ejs

```diff
@@ -0,0 +1,19 @@
+<%
+    var navTrace = new Array();
+
+    if (typeof pageTitle !== 'undefined') {
+        navTrace.push(pageTitle);
+    }
+    // If page is associated with a particular course,
+    // always display the short name, e.g. QA 101
+    if (typeof course !== 'undefined'
+    && typeof course.short_name !== 'undefined') {
```

Comment:
Courses must have a `short_name` so we can skip this check.

Reply:
Fixed in de59867